package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "bookstore-react-redux",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^5.1.0",
+    "react": "^17.0.2",
+    "react-bootstrap": "^2.0.0-beta.5",
```

Comment:
- [ ] As the requirement says you need to add the redux see the screenshot 👇


![b](https://user-images.githubusercontent.com/61562626/130681473-db37ae51-e733-4677-9a5b-3db281539bd8.jpeg)